### ISSUE TITLE
OpenSuse 42.1: semlock broken

### ISSUE THREAD

**Issue by markus2330** [ElektraInitiative/libelektra#1122]
```
SEMLOCK     TESTS
==================

test Open & Close
There are 2 warnings
buffer is: warnings/#00
number: 11
description: open of plugin returned unsuccessfully (reason contains plugin, see other warnings for details)
ingroup: kdb
module:
file: /libelektra/src/libs/elektra/plugin.c
line: 297
reason: semlock
reason:
reason:
buffer is: warnings/#01/configfile
number: (null)
description: kdbClose() failed
ingroup: kdb
module:
file: /libelektra/src/libs/elektra/plugin.c
line: 328
reason: kdbClose() failed
reason:
reason:
buffer is: warnings/#02/configfile
number: (null)
description: (null)
ingroup: (null)
module: (null)
file: (null)
line: (null)
reason: (null)
reason: (null)
reason: (null)

...

buffer is: warnings/#85/configfile
number: (null)
description: (null)
ingroup: (null)
module: (null)
file: (null)
line: (null)
reason: (null)
reason: (null)
reason: (null)
/libelektra/src/plugins/semlock/testmod_semlock.c:23: error in test_OpenClose: warnings in kdbOpen for plugin semlock
number: 145
description: : semlock plugin error
ingroup: : plugin
module: : semlock
at: /libelektra/src/plugins/semlock/semlock.c:58
reason: : Open semaphore: ENOSYS. /dev/shm should be mounted as tempfs. Please look in the README using "kdb info semlock"!

mountpoint: :
configfile: :
/libelektra/src/plugins/semlock/testmod_semlock.c:23: error in test_OpenClose: error in kdbOpen for plugin semlock
/libelektra/src/plugins/semlock/testmod_semlock.c:23: fatal in test_OpenClose: could not open semlock plugin
```

**Comment by markus2330** [comment]
`/dev` is not mounted, nevertheless it looks like there are bugs.

**Comment by KurtMi** [comment]
closed due to semlock removal #2411
